builder/builder-next/controller.go

```diff
@@ -135,7 +135,7 @@ func newSnapshotterController(ctx context.Context, rt http.RoundTripper, opt Opt
 		SessionManager:   opt.SessionManager,
 		WorkerController: wc,
 		Frontends:        frontends,
-		CacheKeyStorage:  cacheStorage,
+		CacheManager:     solver.NewCacheManager(context.TODO(), "local", cacheStorage, worker.NewCacheResultStorage(wc)),
```

Comment:
Any reason we don't use `ctx` here? Same question for the same line in `newGraphDriverController`

Reply:
This is based on buildkitd not passing the context here; cc @crazy-max

Reply:
Looking closer, I don't see any reason **not** to plumb it through, so I've gone ahead and done it 😇